sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/CertificateAsyncClient.java

```diff
@@ -269,21 +275,21 @@ private CertificatePolicy getDefaultPolicy() {
      * Gets information about the latest version of the specified certificate. This operation requires the certificates/get permission.
      *
      * <p><strong>Code Samples</strong></p>
-     * <p>Gets a specific version of the key in the key vault. Prints out the
+     * <p>Gets a specific version of the certificate in the key vault. Prints out the
```

Comment:
This could be a little misleading. We don't get the version of the certificate.
It could be reworder to may be `Gets a certificate of the specified version ...`